packages/ui/src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,32 @@
+export interface ProfileCardProps {
+  name: string;
+  imageSrc?: string;
+}
+
+export const ProfileCard = ({ name, imageSrc }: ProfileCardProps) => {
+  const [firstName, middleName, lastName] = name.split(/\s+/);
+  return (
+    <div
```

Comment:
@Shiyamal-Kumar , no need of another div for onClick(), you can add the onClick() to the inner div itself